h2o-docs/src/product/data-science/permutationVarImp.rst

```diff
@@ -0,0 +1,70 @@
+Permutation Variable Importance
```

Comment:
You should also write something about OAT and add an example of usage. Just because It is already implemented and it will be available in H2O by this PR.